templates/core/terraform/variables.tf

```diff
@@ -135,13 +135,18 @@ variable "stateful_resources_locked" {
   description = "Used to add locks on resources with state"
 }
 
-
 variable "ci_git_ref" {
   default     = ""
   description = "The git ref used by the ci to deploy this TRE"
   type        = string
 }
 
+variable "enable_local_debugging" {
+  default     = false
+  description = "This will allow Cosmos to be accesible from your local IP address and add some extra role permissions."
```

Comment:
nit: `accesible` should be `accessible`. If you're fixing this, suggest change `Cosmos` to `CosmosDB`